client/my-sites/sharing/style.scss

```diff
@@ -900,24 +900,24 @@
 	}
 }
 
-@include sharing-button-service( 'facebook', $color-facebook );
-@include sharing-button-service( 'twitter', $color-twitter );
+@include sharing-button-service( 'facebook', var( --color-facebook-rgb ));
+@include sharing-button-service( 'twitter', var( --color-twitter-rgb ));
 @include sharing-button-service( 'press-this', $blue-wordpress );
```

Comment:
this could use `var( --color-primary-rgb )` instead of `$blue-wordpress`, correct?

Reply:
Yep, added a commit for that. :)

Reply:
@torres126 Press This is a WordPress brand so it should use the new --color-wpcom variable, please. In other words, we don't want the color to change with schemes, which --color-primary might do.

Why are these rgb by the way?

Reply:
Oh I see you all discussed it above.